package.json

```diff
@@ -4,15 +4,17 @@
   "private": true,
   "main": "expo-router/entry",
   "scripts": {
-    "start": "expo start",
+    "start": "expo start -c",
```

Comment:
We really need to clear the bundler cache at every start ?

Reply:
yes, otherwie sometimes it will not detect about the change of the env variable, i tested that and i fixed it by using -c